org.munadi.Munadi.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.munadi.Munadi",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "munadi",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc", 
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-config/autostart:create",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "munadi",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 org.munadi.Munadi.desktop /app/share/applications/org.munadi.Munadi.desktop",
+                "install -Dm644 org.munadi.Munadi.appdata.xml /app/share/appdata/org.munadi.Munadi.appdata.xml",
+                "tar xf org.munadi.Munadi.icons.tar.gz -C /app/share/",
+                "mv src/* .",
+                "qmake munadi.pro",
+                "make -j4",
```

Comment:
It's better to use `-j $FLATPAK_BUILDER_N_JOBS` instead of `-j4`.

Reply:
Thanks, will do.